### ISSUE TITLE
AndroidTV: Accessibility Server DPAD Key Events Handler

### ISSUE THREAD

**Issue by DavidFlores24** [react-native-webview/react-native-webview#1882]
**Problem**
`react-native-webview` is currently incompatible with the TalkBack feature on AndroidTV.
As of now, any DPAD Key Events are being consumed by Android's Accessibility Server making navigation within the application impossible.

Any DPAD Key Events captured by the application behave similarly to a `tab` keypress on a keyboard and a green box shows the current focus of the application. However, the application inside the webview should treat DPAD keypresses 
differently depending on the direction of the keypress.

In short, the TalkBack navigation and the webview navigation are not in sync.

Would it be possible to enable a feature that bubbles the DPAD key events to both TalkBack & the webview content?

More info:
https://developer.android.com/training/tv/accessibility/non-native-app#why_the_dpad_key_events_are_being_consumed

**Additional context**
Tech Stack
react-native: `v0.59.9`
react-native-webview: `v^6.8.0`

Alternatively, any help on how to bypass this would be greatly appreciated.
